packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -36,6 +36,16 @@ const useStyles = makeStyles<Theme, GQLReferenceWidgetStyle>((theme) => ({
     fontWeight: ({ bold }) => (bold ? 'bold' : null),
     textDecorationLine: ({ underline, strikeThrough }) => getTextDecorationLineValue(underline, strikeThrough),
   },
+  endAdornmentButton: {
+    position: 'absolute',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    display: 'flex',
+    right: theme.spacing(2.5),
+    '& > *': {
+      padding: 0,
+    },
```

Comment:
The absolute positioning feels odd but why do you need to position things using percentages, translation, fixed size in pixels simultaneously?

Reply:
With the last changes, I don't need the top and transform properties, I will change that.
For the rigth, it corresponds to the offset for the actual arrow button that came with the Autocomplete mui component.